t/query/country_cities.rb

```diff
@@ -29,4 +29,22 @@
       end
     end
   end
+
+  describe 'United States of America (Q30)' do
+    around { |test| VCR.use_cassette('CountryCities Q30', &test) }
+
+    let(:cities) { Query::CountryCities.new(id: 'Q30').data }
+
+    describe 'Los Angeles' do
+      subject { cities.find { |c| c.id == 'Q65' } }
+
+      it 'should only list Los Angeles once' do
+        cities.select { |c| c.name == 'Los Angeles' }.count.must_equal 1
+      end
```

Comment:
The logic here is still a little odd for my brain. As implemented it looks as if we actively care about the difference in filtering by `id` vs filtering by `name`, but I don't think that's what's really being tested here, is it?

Reply:
Thanks for pointing this out - it does look a little weird. It's [about to be removed](https://github.com/everypolitician/legislative-explorer/pull/23/files#r168360618) which means it will no longer matter for this PR but I'll keep that in mind for future changes 👍